pages/styles/about.css

```diff
@@ -0,0 +1,57 @@
+.container {
+    background: rgb(54, 52, 52);
+    color: rgb(235, 232, 221);
+}
+
+header {
+    font-family: cursive;
+    font-size: 18pt;
+    color: rgb(204, 236, 204);
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+}
+
+nav {
+    color: rgb(163, 230, 197);
+    font: Helvetica;
+    height: 5em;
+    display: flex;
+    align-items: center;
+    justify-content: center 
+}
+
+ul {
+    list-style-type: none;
+}
+
+.section {
+    display: flex;
+    flex-wrap: wrap;
+    align-items: center;
+}
+
+img {
+    display: block;
```

Comment:
There's no need to set the display to block (and in fact, this is probably being overwritten since the img tags are placed under a flex display layout item). Any kind of tag, regardless of its intrinsic display type, will become a flex child when nested under a display flex layout item.